var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -345,7 +345,8 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage):
     depends_on("cudnn@:6", when="@0.5:0.6 +cuda")
     depends_on("cudnn@:7", when="@0.7:2.2 +cuda")
     # depends_on('tensorrt', when='+tensorrt')
-    depends_on("nccl", when="+nccl")
+    depends_on("nccl", when="+nccl+cuda")
+    depends_on("rccl" when="+nccl")
```

Comment:
I think you'll want to modify the existing `depends_on("rccl")` line, not this one

Reply:
Makes sense.  Fixed.  Sorry about that.